corehq/apps/reports/views.py

```diff
@@ -1104,6 +1120,22 @@ def _render_report_configs(request, configs, domain, owner_id, couch_user, email
     return response.content.decode("utf-8"), excel_attachments
 
 
+def _is_location_restricted(domain, config, couch_user):
+    '''
+    Checks if request is location restricted for the asked report.
+    Returns True if request is restricted for the asked user and the report. Returns False otherwise.
+    '''
```

Comment:
I think the terminology here is a bit misleading.  Users can be "location restricted", meaning they do not have the "access_all_locations" permission.  Views or reports can be "location safe", meaning they can safely be accessed by restricted users.  I think what this function is checking is whether a config should be accessible to a specific user, so maybe `_is_location_safe` or `_is_accessible` would be a more accurate name.

Reply:
Changed.